### DISCUSSION TITLE
Icons in webversion of VS Code are missing

### DISCUSSION THREAD

**Discussion by maiertech** [community/community#24997] [Codespaces]
<p>Another thing I noticed are missing icons in Codespaces accessed via the web:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/3/f3e7381959f519d1c12cb0cc8cb25c1e019c7482.png" data-download-href="/uploads/short-url/yNFpqaQhQkCge5f6ZpGknz5yH98.png?dl=1" title="Screen Shot 2021-01-19 at 3.27.40 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1482402/181088047-e828034f-74fd-4d07-973b-ef7da46149f9.png" alt="Screen Shot 2021-01-19 at 3.27.40 PM" data-base62-sha1="yNFpqaQhQkCge5f6ZpGknz5yH98" width="353" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f3e7381959f519d1c12cb0cc8cb25c1e019c7482_2_353x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f3e7381959f519d1c12cb0cc8cb25c1e019c7482_2_529x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/3/f3e7381959f519d1c12cb0cc8cb25c1e019c7482.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f3e7381959f519d1c12cb0cc8cb25c1e019c7482_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-01-19 at 3.27.40 PM</span><span class="informations">692×980 49.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jkeech** [comment]
<p><a class="mention" href="/legomushroom">@legomushroom</a>, <a class="mention" href="/plisy">@plisy</a></p>

**Comment by legomushroom** [comment]
<p>We are actively looking into the issue and hope it will be resolved soon :blush:</p>

**Comment by maiertech** [comment, marked as answer]
<p>This issue seems to have disappeared and I am now marking it as resolved.</p>
